api/v1_queues.inc

```diff
@@ -82,28 +82,34 @@ function api_v1_queue_projects($method, $data, $query_params)
     $unheld_only = (bool) get_flag_value($query_params, "unheld_only");
 
     $valid_return_fields = [
```

Comment:
We should either have `fetch_queue_projects_data()` return fields that match the database columns (probably) and generalize them here for the API, or return generalized forms used in the API, but not a mix of the two.

Reply:
OK. Since `fetch_queue_projects_data` is currently renaming some of the not-consistent-with-project-table fields, we may as well do the username -> project_manager rename there too.